src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -114,7 +114,7 @@ protected function configure()
     /**
      * {@inheritdoc}
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
The `ContainerDebugCommand` is marked as `@internal` and all subclasses already introduced an `int` return type-hint for `execute()`. So I think it is save to add a return type here as well.